examples/configs/onnx_infer.yml

```diff
@@ -0,0 +1,82 @@
+# defaults:
+#   - _self_
+#   - base_configparams
+
+task:
+  name: ONNXTask
+  params:
+    path_to_onnx: '/workdir/vpatrushev/small2/resnet18-v2-7.onnx'
+    device_id: 0
+    providers: ['CUDAExecutionProvider']
+    inputs:
+      - shape: [1, 3, &height 224, &width 224]
+        dtype: 'float32'
+  
+
+data:
+  TEST:
+    - dataloader:
+        batch_size: 1
```

Comment:
Can we use a larger batch size? If so, make it larger. Otherwise, a user might think that we only work with `batch_size=1` which isn't ok